src/components/LoginRegister/RegisterInput.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { useApolloClient } from '@apollo/client';
+import { useDispatch } from 'react-redux';
+import { Flex, Input, Button, Heading, VStack } from '@chakra-ui/react';
+import { register } from '../../data/actions/auth';
+
+const RegisterInput = () => {
+  const client = useApolloClient();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [submitState, setSubmitState] = useState('');
+  const dispatch = useDispatch();
+
+  const handleClick = async e => {
+    e.preventDefault();
+    dispatch(register(firstName, lastName, email, password, client)).then(
+      registered => {
+        if (!registered) setSubmitState('FAIL');
+      },
+      error => {
+        setSubmitState('FAIL');
+      },
+    );
+  };
+
+  const onChangeFirstName = e => {
+    setFirstName(e.target.value);
+  };
+
+  const onChangeLastName = e => {
+    setLastName(e.target.value);
+  };
+
+  const onChangeEmail = e => {
+    setEmail(e.target.value);
+  };
+
+  const onChangePass = e => {
+    setPassword(e.target.value);
+  };
+
+  return (
+    <form onSubmit={e => handleClick(e)}>
+      <VStack spacing="24px">
+        <Flex w="100%" justify="left">
```

Comment:
could you make the same changes here as the `LoginInput` (so remove flexbox and use `alignSelf`

Reply:
Done!